src/services/reserve/WeightDriftAuditService.ts

```
@@ -0,0 +1,366 @@
+
+      // Emit audit log for audit creation
+      await auditService.logAuditEntry({
+        eventType: "WEIGHT_DRIFT_AUDIT_CREATED",
+        entityType: "WeightDriftAudit",
+        entityId: auditRecord.id,
+        action: "create",
+        performedBy: createdBy,
+        newValue: {
+          status: "pending",
+          currenciesExceedingThreshold:
+            report.currenciesExceedingThreshold,
```

ESLint

[error] Delete `⏎···········`

(prettier/prettier)